### ISSUE TITLE
(5.1) Crashes when I choose "Modify attacks/keysounds at current beat"

### ISSUE THREAD

**Issue by 1033Forest** [stepmania/stepmania#1233]
Pressed enter to go to all the insert/delete beat options and when I want to add an attack it crashes.

**Comment by kyzentun** [comment]
How about I remove the "Add attack" option, because it's part of the old unsupported mod system?

**Comment by 1033Forest** [comment]
Yeah... but why do the mods in Simply Love's modifier menu still work in 5.1? (the theme is for 5.0 and it works on 5.1, the only think that doesn't work is the noteskin selection)

**Comment by kyzentun** [comment]
Because I put in a couple days of work to make a compatibility mode so that UPSRT could work in 5.1 without being modified.

**Comment by 1033Forest** [comment]
Just remove that add attack option because it doesnt work anymore...

**Comment by DXUltimate** [comment]
I don't think a complete removal of this "feature" should be done considering people use attacks in their simfiles, especially in making Pump It Up Infinity edits (since those .edit files actually support the #ATTACK:; tag), but instead, a reworking of this option (with the previously mentioned "editor" rewrite) to work with the new modding system it uses. Really, this would purely be for convenience sake for the step artists simply because we can just manually add attacks into the .ssc files anyways.

Also, honestly, themes designed for 5.0.x should not be used in 5.1 builds. To say something should be removed because a theme that is not updated yet for the 5.1 branch seems a little farfetched considering the option for this I *think* was actually removed in the default theme (but code still existing for it), so unless if there was modification in the default theme, it would be impossible to reproduce the issue otherwise without the option being there in the first place. Not to mention, as Kyzentun previously stated he worked in a compatibility mode for UPSRT that potentially allowed the theme to run during it without additional modification, so if this wasn't done, I don't think you would be able to run the theme without excess spam of issues between select screens or crashes.
